### PR TITLE
fix: profile tabs

### PR DESCRIPTION
**Pull request by ice-brontes**
## Description
1. Update image for reporting user dialog.
2. Update texts/icons for empty state of the profile tabs

## Type of Change
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change
- [x] Refactoring
- [ ] Documentation
- [ ] Chore

## Screenshots (if applicable)
https://github.com/user-attachments/assets/23de9c75-245e-4198-8ceb-1a6b77919a9d

### CODE REVIEW

**Review by ice-kreios — DISMISSED** [review]

**Review by ice-ajax — APPROVED** [review]

File: assets/svg/wallet_icon_wallet_report.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ice-alcides) [inline assets/svg/wallet_icon_wallet_report.svg:1]: is this correct size of SVG icon?

File: lib/app/features/user/providers/tab_data_source_family.c.dart [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:ion/app/features/nostr/providers/entities_paged_data_provider.c.dart';
+import 'package:ion/app/features/user/model/tab_entity_type.dart';
+import 'package:ion/app/features/user/providers/user_articles_data_source_provider.c.dart';
+import 'package:ion/app/features/user/providers/user_posts_data_source_provider.c.dart';
+import 'package:ion/app/features/user/providers/user_replies_data_source_provider.c.dart';
+import 'package:ion/app/features/user/providers/user_videos_data_source_provider.c.dart';
+
+final tabDataSourceProviderFamily =
```
Comment (ice-alcides) [inline lib/app/features/user/providers/tab_data_source_family.c.dart:11]: Let's change this provider on the generated one to align with one style for the providers used in the project

File: lib/app/features/user/pages/profile_page/components/tabs/tab_entities_list.dart [hunk 3]
```diff
@@ -3,25 +3,50 @@
...
+  }) {
+    return TabEntitiesList._(
+      type: type,
+      pubkey: pubkey,
+      builder: type == TabEntityType.replies
+          ? (entities) => EntitiesList(entities: entities.toList(), showParent: true)
+          : null,
+      key: key,
+    );
+  }
```
Comment (ice-endymion) [inline lib/app/features/user/pages/profile_page/components/tabs/tab_entities_list.dart:31]: Maybe better create separate factory constructor for replies? ` factory TabEntitiesList.replies`

File: lib/app/features/user/providers/tab_data_source_family.c.dart (hunk 2 above)
Comment (ice-brontes) [reply lib/app/features/user/providers/tab_data_source_family.c.dart]: Done.

File: lib/app/features/user/pages/profile_page/components/tabs/tab_entities_list.dart (hunk 3 above)
Comment (ice-brontes) [reply lib/app/features/user/pages/profile_page/components/tabs/tab_entities_list.dart]: Done, good idea.

File: assets/svg/wallet_icon_wallet_report.svg (hunk 1 above)
Comment (ice-brontes) [reply assets/svg/wallet_icon_wallet_report.svg:1]: Rechecked the icon, looks like this size is correct.

**Review by ice-endymion — APPROVED** [review]

**Review by ice-alcides — APPROVED** [review]

**Review by ice-kreios — APPROVED** [review]

**Pull request merged**